Add string metric to Glean docs [ci skip] (#2543)
mozilla-mobile/android-components: components/service/glean/docs/metrics/adding-new-metrics.md

    ...
```

> Now you can use the counter from the applications code:

Reviewer: ```suggestion
Now you can use the counter from the application's code:
```

assertEquals(6, Controls.refreshPressed.testGetValue())
```

## Strings

> Used to record a single string value, say the name of the default search engine in a browser.

Reviewer: ```suggestion
Used to record a single string value, such as the name of the default search engine in a browser.
```

Say you're adding a metric to find out what the default search in a browser is. First you need to add an entry for the metric to the `metrics.yaml` file:

```YAML
search.default:
  name:
    type: string
    description: >
      The name of the default search engine.
    ...
```

> Now you can use the string from the applications code:

Reviewer: ```suggestion
Now you can use the string from the application's code:
```

```Kotlin
import org.mozilla.yourApplication.GleanMetrics.SearchDefault

// Record a value into the metric.
SearchDefault.name.set("duck duck go")
// If it changed later, you can record the new value:
SearchDefault.name.set("wikipedia")
```

There are test APIs available too:

```Kotlin
import org.mozilla.yourApplication.GleanMetrics.SearchDefault
Glean.enableTestingMode()

// Was anything recorded?
assertTrue(SearchDefault.name.testHasValue())
// Does the string metric have the expected value?

> assertEquals("wikipedia", SearchDefault.name.testGetValue())

Reviewer 2: We need to close the code fragment here using 3 backticks, otherwise the rest of the doc won't render properly.
